test/governance/InterchainGovernance.js

```diff
@@ -386,9 +380,9 @@ describe('InterchainGovernance', () => {
         const bytecodeHash = keccak256(bytecode);
 
         const expected = {
-            istanbul: '0x2534d1533c9ffce84d3174c1f846a4041d07b56d1e7b5cb7138e06fb42086325',
-            berlin: '0x1084d7de843267ed6f4ad87cbdc541bfb2aa003c67c285d0b4f90b3026370f7e',
-            london: '0x9d89dce5b3087d6f9a1b80cc3e96ae9c204a1ce3c2c4eb5bce7671a20a635f97',
+            istanbul: '0x54b6df21db64966ecbce299195346d2e296ad799a4187460afe2d22c9c4167d6',
+            berlin: '0x637c08b2a9f8161b430e96859e4799728a6279ddfc82fb68e863d0ea895908fb',
```

Comment:
we can drop the older versions. we don't test those anymore

Reply:
got it. updated.